### ISSUE TITLE
Questions about dataset statistics

### ISSUE THREAD

**Issue by TLB-MISS** [omniobject3d/OmniObject3D#5]
First of all, thank you so much for providing such a great dataset.
I have minor questions about dataset statistics.
The paper states that the dataset provides a total of 6000 instances in 190 categories.
However, when I downloaded the dataset, there were a total of 5385 instances in 216 categories.
Having problems with my download? Or is it correct that the number of instances is less than the paper?

Thank you

**Comment by wutong16** [comment]
Hi, thank you for your attention to the paper. 

Regarding the instance number, as stated in the readme, we have reserved 10% of the instances as a "hidden" test set for an upcoming challenge focused on sparse-view nerf and 3D generation. We plan to release the codebase and available sparse views for these test sets later this month. However, the final version of the test data may not be as large as initially indicated. The remaining instances will be merged with the current training set and released too in this month.

In terms of the category number, it is indeed subject to change due to recent efforts to divide certain categories into more fine-grained subcategories.

**Comment by TLB-MISS** [comment]
@wutong16 
Ah, sorry. I should have read the documentation more carefully. Thanks for the kind reply!
